setup.py

```diff
@@ -57,8 +57,8 @@
         'threadloop>=1,<2',
 
         # tracing deps
-        'opentracing>=1.1,<2',
-        'opentracing_instrumentation>=2.3,<3',
+        'opentracing>2',
+        'opentracing_instrumentation>3',
```

Comment:
so…this now conflicts with jaeger-client-python, which requires opentracing_instrumentation<3

Reply:
jaeger-client-python is internal to Uber so lets take this convo to uChat :)